# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: architecture/loongarch64/package/*/hotfix-mount.h.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-5.4/include/uapi/linux/mount.h.vanilla	2022-11-01 13:00:35.615893986 +0100
+++ linux-5.4/include/uapi/linux/mount.h	2022-11-01 13:00:50.726892413 +0100
@@ -85,6 +85,8 @@
 #define FSPICK_NO_AUTOMOUNT	0x00000004
 #define FSPICK_EMPTY_PATH	0x00000008
 
+#if 0
+
 /*
  * The type of fsconfig() call made.
  */
@@ -140,4 +138,6 @@
 /* List of all mount_attr versions. */
 #define MOUNT_ATTR_SIZE_VER0	32 /* sizeof first published struct */
 
+#endif
+
 #endif /* _UAPI_LINUX_MOUNT_H */
